test/RateTargetKinkInterestRateStrategy.t.sol

```diff
@@ -72,7 +72,8 @@ contract RateTargetKinkInterestRateStrategyTest is InterestRateStrategyBaseTest
         assertEq(interestStrategy.getBaseVariableBorrowRate(), 0.01e27);
         assertEq(interestStrategy.getMaxVariableBorrowRate(),  0.595e27);
 
-        rateSource.setRate(0.07e27);
+        rateSource.setRate(0.07e18);
+        rateSource.setDecimals(18);
```

Comment:
Same here lets put this in a dedicated test

Reply:
done